.github/workflows/migrator.yml

```diff
@@ -295,20 +295,25 @@ jobs:
           git config --global user.email "${{ secrets.PYANSYS_CI_BOT_EMAIL}}"
           git config pull.rebase true
 
-          echo "Adding ${{ env.PR_HEAD_REPO }} as remote"
+          echo "\033[1;92m[INFO]Adding ${{ env.PR_HEAD_REPO }} as remote \033[0m"
           git remote add head_repo https://x-access-token:${{ secrets.PYANSYS_CI_BOT_TOKEN }}@github.com/${{ env.PR_HEAD_REPO }}.git
 
-          echo "Fetching '${{ env.PR_HEAD_BRANCH }}' branch from '${{ env.PR_HEAD_REPO }}'"
+          echo "\033[1;92m[INFO]Fetching '${{ env.PR_HEAD_BRANCH }}' branch from '${{ env.PR_HEAD_REPO }}' \033[0m"
           git fetch head_repo ${{ env.PR_HEAD_BRANCH }}
-          git pull head_repo ${{ env.PR_HEAD_BRANCH }}
```

Comment:
Now we dont `pull` after `fetch`... we do `fetch`, `checkout` and `pull`. See later...